Cargo.toml

```diff
@@ -134,18 +133,13 @@ tokio-util = { version = "0.7", features = [
 tokio-stream = { version = "0.1", default-features = false, optional = true }
 walkdir = { version = "2.5", default-features = false, optional = true }
 
-dojo-test-utils = { git = 'https://github.com/dojoengine/dojo', tag = "v1.0.0-alpha.0", default-features = false, optional = true }
-katana-primitives = { git = 'https://github.com/dojoengine/dojo', tag = "v1.0.0-alpha.0", default-features = false, features = [
+dojo-test-utils = { git = 'https://github.com/dojoengine/dojo', rev = "1d3318a", default-features = false, optional = true }
+katana-primitives = { git = 'https://github.com/dojoengine/dojo', rev = "1d3318a", default-features = false, features = [
```

Comment:
can we set a tag? `v1.0.0-alpha.1`?

Reply:
This was not possible at the time I made the PR because the interesting commit was right after the last tag at the time (v1.0.0-alpha.1) but it is now possible as a new tag has been released in the meantime (v1.0.0-alpha.2).

So I've just updated.